UM/Qt/qml/UM/qmldir

```diff
@@ -25,5 +25,6 @@ ProgressBar 1.3 ProgressBar.qml
 SimpleButton 1.1 SimpleButton.qml
 TabRow 1.3 TabRow.qml
 TabRowButton 1.3 TabRowButton.qml
+CheckBox 1.0 CheckBox.qml
```

Comment:
```suggestion
CheckBox 1.5 CheckBox.qml
```
Well, no? You can't suddenly add a new type to a lower semver. We haven't been consistent with this, I do admit that, but today is a great day to start :)